app/ui.js

```diff
@@ -369,9 +369,9 @@ var UI;
                 case 'connected':
                     UI.connected = true;
                     if (rfb && rfb.get_encrypt()) {
-                        msg = "Connected (encrypted) to " + UI.desktopName;
+                        msg = Util.Localisation.get("Connected (encrypted) to ") + UI.desktopName;
```

Comment:
A good start perhaps, but this won't work long term. What if the order of words needs to be changed? A printf() style API is need to handle languages outside the Germanic family.